lib/VMwareWebService/MiqVimVm.rb

```diff
@@ -751,7 +751,7 @@ def addDisk(backingFile, sizeInMB, label=nil, summary=nil)
 							# bck.diskMode = VirtualDiskMode::Independent_nonpersistent
 							bck.diskMode		= VirtualDiskMode::Independent_persistent
 						    bck.split			= "false"
-						    bck.thinProvisioned	= "false"
+						    bck.thinProvisioned	= thinProvisioned.to_s
```

Comment:
Any news on this? May be 
`bck.thinProvisioned = (thinProvisioned.to_s.downcase == "true" ? "true" : "false")`
to address all possible options?